CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 3.9)
+cmake_minimum_required(VERSION 3.30)
```

Comment:
3.30 is really really new

Reply:
I've made that change in GTSAM and was going to push it along with some other minor fixes.

(CMake had been yelling at me a lot recently)

Reply:
> Main project is at 3.9 right now [borglab/gtsam@`develop`/CMakeLists.txt](https://github.com/borglab/gtsam/blob/develop/CMakeLists.txt?rgh-link-date=2025-11-22T23%3A22%3A33Z)

I made that change locally, I haven't pushed it yet since I am trying to verify dependencies.

Reply:
I don't have the bandwidth to deal with this (IMO trivial) issue, so I dropped the version down to 3.19.2 since that version is when CMake started supporting Apple Silicon.

(I say trivial since I already spent the time compiling and testing compatibility already before making the PR, as a good contributor would)